src/collections/_documentation/platforms/python/default-integrations.md

```diff
@@ -23,6 +23,24 @@ This integration registers with the interpreter's except hook system.  Through t
 any exception that is unhandled will be reported to Sentry automatically.  Exceptions
 raised interactive interpreter sessions will not be reported.
 
+### Options
+
+You can pass the following keyword arguments to `ExcepthookIntegration()`:
+
+* `always_run`:
+    
+    ```bash
+    $ python
+    >>> import sentry_sdk
+    >>> from sentry_sdk.integrations.excepthook import ExcepthookIntegration
+    >>> sentry_sdk.init(..., integrations=[ExcepthookIntegration(always_run=True)])
+    >>> raise Exception("I will become an error")
+    ```
+    By default, the SDK does not capture errors occuring in the REPL (`always_run=False`).
+
+  The default is `False`.
```

Comment:
This line duplicates information. Also please check the indentation level, I think the code block is indented with three spaces